frontend/src/features/projects/common/hooks/useProject.tsx

```diff
@@ -3,9 +3,21 @@ import { RootState } from 'reducers/rootReducer';
 import { IStatus, IProjectWrapper } from '..';
 import _ from 'lodash';
 import { useHistory } from 'react-router-dom';
+import { initialValues } from '../interfaces';
+
+const getStatusTransitionWorkflow = (
```

Comment:
completes functionality required to submit status updates to new workflows without hardcoding values on the frontend.

Reply:
Now this is neat